cli/generators/component/templates/component-container.js

```diff
@@ -0,0 +1,39 @@
+import React, {PropTypes} from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+
+import * as actions from '../../actions/---AUTOGENERATED---';
```

Comment:
Ideas on a better way to handle this? I couldn't think a quick solution. @juampick